### PR TITLE
Fixes #129 

### LINKED ISSUE #129: Mobile Navigation on documentation should use the same code/design as nav on centrifuge.io
The mobile navigation on developer.centrifuge.io and centrifuge.io should look the same. It should cover the entire screen width instead of just the right part and have the same text spacing for the nav items. We have a styled ui components library at github.com/centrifuge/axis. Both the docs and the website should use a component from axis for the library.

Acceptance Criteria:

- [ ] Modified the developer.centrifuge.io and [website](https://github.com/centrifuge/website) repo to both use a navbar component import from [axis](github.com/centrifuge/axis)

This issue has a gitcoin bounty attached to it. We encourage bounty hunters to join our slack and talk to us. We also encourage you to tackle more than one of our open bounties! If you'd like to start working on it, feel free to reach out to me (lucas@centrifuge.io) directly, join our slack channel `#bounties` on our [slack](https://centrifuge.io/slack/)

### PR DESCRIPTION
**Pull request by acolytec3**
Fixes #129 

Replace existing navbar with navbar component from [Axis](https://github.com/centrifuge/axis-nav-bar

### CODE REVIEW

**Comment by lucasvo** [comment]
Thanks for submitting a WiP PR, the code looks good.  Ping me when you've fixed the TODOs and completed the rest of it and I'll give it a more thorough review.

**Comment by acolytec3** [comment]
I've added the search box from the current page in so all the components are now there.  Where I'm stuck is this infinite render bug and menu auto-closing bugs.  I discovered that if I try a search using the inline search box, the menu then begins working as expected.  I'm at a loss as to why either of the bugs is occurring and was curious if you have any insights?

**Comment by acolytec3** [comment]
@lucasvo I'm still seeing the issue where the menu won't stay open on mobile.  It seems like somehow the `closeMenu` state hook method is being called or else the component is getting reset to it's initial state when the menu button is tapped.

**Comment by acolytec3** [comment]
@lucasvo I fixed the last bug.  This now ready for review.  I'll work on finishing up the matching PR for the main website repo.

**Comment by lucasvo** [comment]
@acolytec3 the rendering is now broken, if you look at it in firefox on a 13" screen it looks like this:
Screen Shot 2020-03-05 at 20 52 06

**Comment by acolytec3** [comment]
> @acolytec3 the rendering is now broken, if you look at it in firefox on a 13" screen it looks like this:
> Screen Shot 2020-03-05 at 20 52 06

To confirm on the part that's broken, I'm assuming you want the search bar to be in line with the navlinks as in the current page, right?

**Comment by lucasvo** [comment]
@acolytec3  
> To confirm on the part that's broken, I'm assuming you want the search bar to be in line with the navlinks as in the current page, right?
yes, the menu should have the full width of the page, right now it's only as wide as the content and the search bar should be in the same line.

**Comment by acolytec3** [comment]
@lucasvo Is this what you're thinking the expanded menu should look like in the mobile view?  I had to modify the navbar component to make it work as the width of the mobile navbar is hard-coded to 70% of the view width [here](https://github.com/centrifuge/axis/blob/ad2f246163fa36309633dcfe0bcb7e6241f3b9e6/packages/nav-bar/src/NavBar.tsx#L215) so would need to update that component as well if this is what you are thinking.

expanded

**Comment by lucasvo** [comment]
Have you looked at the navbar on centrifuge.io? I think the navbar works correctly there.

**Comment by acolytec3** [comment]
Is the goal to have the navbar component on both sites be shared and the visual design/layout/behavior of it match the current centrifuge.io look and feel?  I want to sure I understand what the actual intended outcome of this work is since the original bounty states that the two sites should use the same component and seems to imply that I should just use the navbar component from the axis repo as styled by the repo.  

The navbar component as currently coded won't work on the main website since the current design doesn't group secondary items under their parents and doesn't provide the dropdown feature that's coded for the current main site.

File: src/components/Nav/navbar/NavBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+describe("Nav Bar test", () => {
```
Comment (lucasvo) [inline src/components/Nav/navbar/NavBar.test.tsx]: Can you delete this file?

**Pull request merged**